unit_test/sparse/Test_Sparse_spadd.hpp

```diff
@@ -52,7 +52,8 @@ crsMat_t randomMatrix(ordinal_type nrows, ordinal_type ncols,
   typename scalar_view_t::HostMirror h_values =
       Kokkos::create_mirror_view(values);
   for (size_type i = 0; i < nnz; i++) {
-    h_values(i) = KAT::one() * (((typename KAT::mag_type)rand()) / RAND_MAX);
+    h_values(i) = KAT::one() * (((typename KAT::mag_type)rand()) /
```

Comment:
I would change for `static_cast<typename KAT::mag_type>` this way both operand have consistent type.